src/pages/About.vue

```diff
@@ -1,15 +1,132 @@
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="section card">
+      <div class="container">
+        <div class="columns is-vcentered">
+          <div class="column is-half">
+            <transition name="fade" mode="out-in">
+              <img
+                :src="svc_img"
+                :key="svc_img"
+                data-toggle="tooltip"
+                data-placement="bottom"
+                title="goTop"
+                alt=""
+              />
+            </transition>
+          </div>
+          <div class="column is-half">
+            <div class="columns is-centered is-vcentered">
+              <div class="column is-three-quarters">
+                <div class="content">
+                  <h2 class="title is-2 has-text-centered">
+                    Services We Provide
+                  </h2>
+
+                  <p
+                    class="button card box is-white is-medium active-card"
+                    @click="mouseOverChangeImage(0, '/assets/img/cloud.jpg')"
+                  >
+                    Web &amp; Email Hosting
+                  </p>
+                  <p
+                    class="button card box is-white is-medium"
+                    @click="
+                      mouseOverChangeImage(1, '/assets/img/unix-shell.jpg')
+                    "
+                  >
+                    <strong>UNIX Shell Accounts</strong>
+                  </p>
+                  <p
+                    class="button card box is-white is-medium"
+                    @click="mouseOverChangeImage(2, '/assets/img/printer.jpg')"
+                  >
+                    <strong>Free Printing for Members</strong>
+                  </p>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
       </div>
     </section>
+    <section class="section card">
+      <div class="container">
+        <div class="columns is-vcentered">
+          <div class="column is-half">
+            <img
+              src="http://3.bp.blogspot.com/-l5WiG6Nci34/UVKfPBediVI/AAAAAAAAABk/48mdp61vr5k/s1600/Emperor-Penguins-Couple.jpg"
+              class="card"
+            />
+          </div>
+          <div class="column is-half">
+            <div class="columns is-centered is-vcentered">
+              <div class="column is-three-quarters">
+                <h2 class="title is-2 has-text-centered">
+                  Joining the OCF is an awesome way to learn!
+                </h2>
+                <h4 class="title is-4">
+                  We love talking tech and learning new things.
+                </h4>
+                <p>
+                  We use our weekly board meetings and staff hours not just to
+                  make decisions, but also to learn more about *nix and free
+                  software.
+                </p>
+                <h4 class="title is-4">
+                  We're totally open and super-inclusive!
```

Comment:
😐

Reply:
What's up?

Reply:
@bjb568 not the place to put snide remarks, a new staffer is translating https://www.ocf.berkeley.edu/about/staff onto the new site.

Reply:
@BernardZhao This text is new.